templates/getting_started_python/.gitignore

```diff
@@ -0,0 +1,8 @@
+# This file tells Git which files shouldn't be added to source control
```

Comment:
Looks quite slim compared to https://github.com/github/gitignore/blob/main/Python.gitignore

Reply:
Good point, I'll make it a bit fatter